packages/ovm/src/app/utils.ts

```diff
@@ -71,24 +71,26 @@ export interface OvmTransactionMetadata {
  */
 export const convertInternalLogsToOvmLogs = (logs: Log[]): Log[] => {
   let activeContract = logs[0] ? logs[0].address : ZERO_ADDRESS
+  const loggerLogs = []
   const ovmLogs = []
   logs.forEach((log) => {
     const executionManagerLog = executionManagerInterface.parseLog(log)
     if (executionManagerLog) {
       if (executionManagerLog.name === 'ActiveContract') {
         activeContract = executionManagerLog.values['_activeContract']
       } else {
-        logger.debug(
+        loggerLogs.push(
           `${executionManagerLog.name}, values: ${JSON.stringify(
             executionManagerLog.values
           )}`
         )
       }
     } else {
-      logger.debug(`Non-EM log: ${JSON.stringify(log)}`)
+      loggerLogs.push(`Non-EM log: ${JSON.stringify(log)}`)
       ovmLogs.push({ ...log, address: activeContract })
     }
   })
+  logger.debug(loggerLogs.join("\\n"))
```

Comment:
Re: suggested change above, maybe expose a const? Other option is to add new lines only to have them removed, but that seems silly 😆